tgui/packages/tgui/interfaces/common/CrewManifest.js

```diff
@@ -0,0 +1,194 @@
+import { useBackend } from "../../backend";
+import { Box, Section, Table } from "../../components";
+import { decodeHtmlEntities } from 'common/string';
+import { COLORS } from "../../constants";
+
+export const CrewManifest = (props, context) => {
+  const { act, data } = useBackend(context);
+  // HOW TO USE THIS THING
+  /*
+  	GLOB.data_core.get_manifest_json()
+	  data["manifest"] = GLOB.PDA_Manifest
+  */
+  // And thats it
+
+
+  const {
+    manifest,
+  } = data;
+
+  const {
+    heads,
+    sec,
+    eng,
+    med,
+    sci,
+    ser,
+    sup,
+    misc,
+  } = manifest;
+
+  const deptCols = COLORS.department;
+
+  const HeadRoles = [
+    "Captain",
+    "Head of Security",
+    "Chief Engineer",
+    "Chief Medical Officer",
+    "Research Director",
+    "Head of Personnel",
+  ];
+
+  // Head colour check. Abbreviated to save on 80 char
+  const HCC = role => {
+    // Return green if they are the head
+    if (HeadRoles.indexOf(role) !== -1) {
+      return "green";
+    }
+
+    // Return yellow if its the qm
+    if (role === "Quartermaster") {
+      return "yellow";
+    }
+
+    // Return orange if its a regular person
+    return "orange";
+  };
+
+  // Head bold check. Abbreviated to save on 80 char
+  const HBC = role => {
+    // Return true if they are a head, or a QM
+    if ((HeadRoles.indexOf(role) !== -1) || role === "Quartermaster") {
+      return true;
+    } else {
+      return false;
+    }
+  };
+
+  const ManifestTable = group => {
+    return (
+      group.length > 0 && (
+        <Table>
+          <Table.Row header color="white">
+            <Table.Cell width="50%">Name</Table.Cell>
+            <Table.Cell width="35%">Rank</Table.Cell>
+            <Table.Cell width="15%">Active</Table.Cell>
+          </Table.Row>
+
+          {group.map(person => (
+            <Table.Row
+              color={HCC(person.rank)}
+              key={person.name + person.rank}
+              bold={HBC(person.rank)}>
+              <Table.Cell>{decodeHtmlEntities(person.name)}</Table.Cell>
+              <Table.Cell>{decodeHtmlEntities(person.rank)}</Table.Cell>
+              <Table.Cell>{person.active}</Table.Cell>
+            </Table.Row>
+          ))}
+        </Table>
+      )
+    );
+  };
+
+  return (
+    <Box>
+      <Section
+        title={(
+          <Box backgroundColor={deptCols.command} m={-1} pt={1} pb={1}>
+            <Box ml={1} textAlign="center" fontSize={1.4}>
+              Command
+            </Box>
+          </Box>
+        )}
+        level={2}>
+        {ManifestTable(heads)}
+      </Section>
```

Comment:
There's a lot of hard coding in there and I don't like it. Might be good to simplify everything into an `Array` and map through it with one `Section` per line.